database/index.js

```diff
@@ -1,26 +1,69 @@
-const Sequelize = require('sequelize');
+const { Pool } = require('pg');
 const dotenv = require('dotenv');
-const models = require('./models');
 dotenv.config();
 
-const sequelize = new Sequelize('reviews', process.env.DB_USERNAME, process.env.DB_PASSWORD, {
-  host: 'localhost',
-  dialect: 'mysql'
-});
+const pool = new Pool({
+  host: process.env.HOST,
+  user: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: 'steam_reviews'
+})
+
 
-var getReviews = async (GameId) => {
-  const reviews = await models.Review.findAll({
-    include: [{
-      model: models.User,
-    }, {
-      model: models.Game,
-    }, {
-      model: models.User_game,
-    }],
-    where: { GameId },
-  });
-  return reviews;
-};
+
+var getReviews = (gameId, callback) => {
+  pool.connect((err, client, release) => {
+    if (err) {
+      callback(err.stack);
+    }
+    client.query(
+      `select
+      reviews.id,
```

Comment:
is there a reason to avoid `*`?

Reply:
I avoided the use of * because of the use of json_build_objects. Didn't want the data to come in as a flat row, so needed to specify which data points would be where.